### PR TITLE
c2c template added

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/contributing/templates/c2c-source.md [hunk 1]
```diff
@@ -1,110 +1,139 @@
...
+tags:
+  - cloud-to-cloud
+  - c2c-source  #example: Druva
+description:  Description goes here. #example: Learn how to configure the Druva Cloud-to-Cloud Source in your Sumo Logic environment.
 ---
 
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/img/c2c/dropbox/example.json';
+import MyComponentSource from '!!raw-loader!/img/c2c/dropbox/example.json';
+import TerraformExample from '!!raw-loader!/img/c2c/dropbox/example.tf';
```
Comment (jpipkin1) [inline docs/contributing/templates/c2c-source.md:16]: @JV0812, these lines are causing the following error when I attempt to build locally:
> ERROR in ./docs/contributing/templates/c2c-source.md 102:0-75
Module not found: Error: Can't resolve 'raw-loader' in '/Users/jpipkin/Documents/GitHub/sumologic-documentation/docs/contributing/templates'

Comment (JV0812) [reply docs/contributing/templates/c2c-source.md]: Thanks @jpipkin1. I have fixed this now.

File: docs/contributing/templates/c2c-source.md [hunk 2]
```diff
@@ -1,110 +1,139 @@
...
-Sumo Logic Cloud-to-Cloud (C2C) Sources inform you, in real time, about Source health, start-up progress, and errors requiring user action (such as connection failures).
+Example:
+| Polling Interval | Data |
+| --- | --- |
+| 5 min |  [Team Events](https://www.dropbox.com/developers/documentation/http/teams#team_log-get_events)
+-->
 
-C2C Sources go through the following states when created:
+| Polling Interval | Data |
+| --- | --- |
```
Comment (kimsauce) [inline docs/contributing/templates/c2c-source.md:50]:
```suggestion
| :--- | :--- |
```

File: docs/contributing/templates/c2c-source.md [hunk 3]
```diff
@@ -1,110 +1,139 @@
...
-Insert steps to configure on source's website
+## Metadata fields
 
-Example: https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-identity-protection-source/#set-up-a-service-application
+<!--
+Insert meta deta fields in the Sumo Logic UI. Update the below table accordingly.
 -->
 
+| Field | Value | Description |
+| --- | --- | --- |
```
Comment (kimsauce) [inline docs/contributing/templates/c2c-source.md:86]:
```suggestion
| :--- | :--- | :--- |
```

File: docs/contributing/templates/c2c-source.md [hunk 4]
```diff
@@ -1,110 +1,139 @@
...
+| category | String | No | `null` | Type a category of the source. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_sourceCategory`. See [best practices](/docs/send-data/best-practices) for details. | `"mySource/test"`
+| fields | JSON Object | No | `null` | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field _siemForward to enable forwarding to SIEM.|`{"_siemForward": false, "fieldA": "valueA"}` |
+| {{`Parameter`}} | {{Type}} | {{Yes/No}} | `null` | {{Description}} |  {{Example}} |
 
-### Restarting your Source
+### JSON example
 
 <!--
-Paste this line: {@import ../../reuse/restart-c2c-source.md}
+Create and Add the JSON config in the dropbox and import it here.
```
Comment (kimsauce) [inline docs/contributing/templates/c2c-source.md:116]:
```suggestion
Create and add the JSON config in the dropbox and import it here.
```

File: docs/contributing/templates/c2c-source.md [hunk 5]
```diff
@@ -1,110 +1,139 @@
...
+[Download example](/img/c2c/c2c-name/example.json) <!--#example(/img/c2c/dropbox/example.json)-->
 
-| Parameter | Type | Required | Description | Access |
-|:--|:--|:--|:--|:--|
-| {{`Parameter`}} | {{Type}} | {{Yes/No}} | {{Description}} |   |
+### Terraform example
 
-### JSON config parameters
+<!--
+Create and Add the Terraform config in the dropbox and import it here.
```
Comment (kimsauce) [inline docs/contributing/templates/c2c-source.md:126]:
```suggestion
Create and add the Terraform config in the dropbox and import it here.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/source-info.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
+---
+id: source-info
+title: Cloud to Cloud Info
+slug: /c2c/info
+tags:
+  - cloud-to-cloud
+---
+## States
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/source-info.md:8]:
```suggestion

## States
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/source-info.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+1. **Pending**. Once the Source is submitted, it is validated, stored, and placed in a **Pending** state.
+1. **Started**. A collection task is created on the Hosted Collector.
+1. **Initialized**. The task configuration is complete in Sumo Logic.
+1. **Authenticated**. The Source is successfully authenticated with the vendor API(s).
+1. **Collecting**. The Source is actively collecting data from the vendor API(s).
+
+If the Source has any issues during any one of these states, it is placed in an **Error** state.
+
+When you delete the Source, it is placed in a **Stopping** state. When it has successfully stopped, it is deleted from your Hosted Collector.
+On the [Collection page](/docs/manage/health-events#collection-page), the Health and Status for Sources is displayed. Use [Health Events](/docs/manage/health-events.md) to investigate issues with collection.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/source-info.md:20]:
```suggestion

On the [Collection page](/docs/manage/health-events#collection-page), the Health and Status for Sources is displayed. Use [Health Events](/docs/manage/health-events.md) to investigate issues with collection.
```

File: docs/contributing/templates/c2c-source.md [hunk 8]
```diff
@@ -1,110 +1,139 @@
 ---
 id: c2c-source  #example: druva-source
 title: XYZ Source #example: Druva Source
+image: 'https://app_icons.s3.amazonaws.com/dropbox.svg' #app logo
```
Comment (kimsauce) [inline docs/contributing/templates/c2c-source.md:4]:
```suggestion
image: 'https://app_icons.s3.amazonaws.com/dropbox.svg' #replace with your app logo
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor fixes

**Review by kimsauce — APPROVED** [review]

**Pull request merged**